### PR TITLE
Add gerablog static blog generator

### PR DESCRIPTION
**Pull request by paulohrpinheiro**
Please, can you add my blog static generator to the list? Thanks.

### CODE REVIEW

File: list.json [hunk 1]
```diff
@@ -149,7 +149,7 @@
     "website": "http://blogcpp.org",
     "license": "WTFPL",
     "language": "C++",
-    "description": "Just another static blog generator, written in C++17.",
+    "description": "Just another generator, written in C++17.",
```
Comment (balupton) [inline list.json]: Why was this changed?

File: list.json [hunk 2]
```diff
@@ -632,6 +632,12 @@
     "github": "samstefan/genesis"
   },
   {
+    "name": "Gerablog",
+    "github": "paulohrpinheiro/gerablog",
+    "license": "MIT",
+    "description": "Gerablog is a static blog generator. Write your posts in in *Markdown*, publish in *HTML*. Just two commands (create and generate) and seven templates (header, footer, categories page, categories block, main index, post and feed)."
+  },
+  {
```
Comment (balupton) [inline list.json]: LGTM

Comment (paulohrpinheiro) [inline list.json]: Oops, I'm sorry. I was looking for if there was any "is" with "static generator". I must have pressed space or something. It was not intentional, again, I'm sorry. I will fix.
